plugin/web/extensions/location/index.html

```diff
@@ -0,0 +1,28 @@
+<title></title>
+<style>
+    html, body{
+        background: transparent;
+    }
+    .root{
+        background: #fff;
+        border-radius: 0 0 8px 0;
+    }
+    html, body, .root {
+        width: 1680px;
```

Comment:
Is this not wayyyyyy too wide?

Reply:
Fixed

Reply:
ca838bb63dff7bc9efdcc1e7ad0376fefc3c3404